Update download-software-licenses-csp.md

msbemba opened the pull request:
Updated instructions for accessing subscription page

msbemba committed:
Update download-software-licenses-csp.md

Updated instructions for accessing subscription page

msbemba commented:
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7231

msbemba commented:
@JohanFreelancer9 Please copy review

JohanFreelancer9 commented on microsoft-365/admin/setup/download-software-licenses-csp.md:
```diff
@@ -34,7 +34,7 @@ You must be a Global admin to do the steps in this article. For more information
 
 1. In the Microsoft 365 admin center, go to the **Billing** > <a href="https://go.microsoft.com/fwlink/p/?linkid=842054" target="_blank">Your products</a> page.
-2. On the **Products** tab, in the **Software** section, select the software that you want to download.
+2. On the **Products** tab, in the **Software** section, click on the product to navigate to the subscription details page.
```
```suggestion
2. On the **Products** tab, in the **Software** section, select the product to navigate to the subscription details page.
```

cmcatee-MSFT replied:
@msbemba Please make the change suggest above for line 37.

JohanFreelancer9 reviewed (changes requested):
Copy review - minor suggestion, please.

msbemba committed:
Update microsoft-365/admin/setup/download-software-licenses-csp.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

msbemba committed:
Update download-software-licenses-csp.md

made changes again as it doesn't work when you select. But only does when you click on the product(Hyperlink)

msbemba commented:
@JohanFreelancer9  It works only when you click and do not select. Not sure if this can be better worded.

JohanFreelancer9 commented on microsoft-365/admin/setup/download-software-licenses-csp.md:
```diff
@@ -34,7 +34,7 @@ You must be a Global admin to do the steps in this article. For more information
 
 1. In the Microsoft 365 admin center, go to the **Billing** > <a href="https://go.microsoft.com/fwlink/p/?linkid=842054" target="_blank">Your products</a> page.
-2. On the **Products** tab, in the **Software** section, select the software that you want to download.
+2. On the **Products** tab, in the **Software** section, Click on the product to navigate to the subscription details page.
```
```suggestion
2. On the **Products** tab, in the **Software** section, click on the product to navigate to the subscription details page.
```

JohanFreelancer9 reviewed (changes requested):
@msbemba, thank you for the explanation. Kindly just commit the suggested change to fix the capitalization. Thanks.

msbemba committed:
Update microsoft-365/admin/setup/download-software-licenses-csp.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

yogkumgit commented:
@cmcatee-MSFT Please merge this PR. Thanks

CarnegieJ commented:
> @msbemba, thank you for the explanation. Kindly just commit the suggested change to fix the capitalization. Thanks.

The suggested change is instructional content and not spelling grammar (e.g. capitalization).

yogkumgit commented:
@msbemba FYA

msbemba committed:
Merge branch 'public' into patch-859

msbemba commented:
@yogkumgit The PR is good to be merged.

msbemba committed:
Update microsoft-365/admin/setup/download-software-licenses-csp.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

msbemba commented:
@cmcatee-MSFT The requested changes have been committed.

yogkumgit commented:
@cmcatee-MSFT Please review and merge this PR. Thanks

msbemba committed:
Update download-software-licenses-csp.md

Minor change to an existing PR

msbemba commented:
@JohanFreelancer9 Please copy review

yogkumgit commented:
@cmcatee-MSFT Please merge this PR. Thanks

tiburd commented:
Closing this PR.
Language complies with Microsoft style guidelines.

cc: @msbemba @yogkumgit
